deploy.cfg

```diff
@@ -58,10 +58,13 @@ method-spec-admin-users = kbaseadmin,kbaseadmin2
 # method-spec-shock-url = 
 # method-spec-shock-user = 
 # method-spec-shock-password = 
-# method-spec-docker-registry = 
+# method-spec-shock-token = 
 endpoint-host=https://ci.kbase.us
 endpoint-base=/services
 
 # Default tag used when tag is not specified in Narrative requests.
 # It should be changed into 'release' in production deployment.
 method-spec-default-tag = dev
+
+# The KBase authorization server url.
+auth-service-url = https://kbase.us/services/authorizatio
```

Comment:
Looks like the default setting of this url is missing the last letter in authorization?